app/webpacker/controllers/cookie-acceptance_controller.js

```diff
@@ -0,0 +1,32 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+
+    static targets = ["overlay"]
+
+    connect() {
+        this.checkforCookie();
+    }
+
+    checkforCookie() {
```

Comment:
I would opt for using a cookie library - doing this manually is verbose and error prone:

https://github.com/js-cookie/js-cookie

Reply:
I disagree with this one, we don't plan to have any additional cookie functionality beyond .... is one set at the moment
so bringing in an external dependency for that is a bit overkill

Reply:
Ok, I don't feel strongly about bringing in a library, but I don't think the for loop is required, why not just:

```
var cookie = document.cookie
if (cookie.includes('GiTBetaCookie=Accepted'))
{
    return;
}
```

You may want to use `indexOf` instead of polyfilling `includes`...

Reply:
theoretically if we used a second cookie value for something on the same domain then we would need to loop them to find the one which contains the value we are looking for.  Plus it's on the client, so the client could edit them.

it already uses indexOf